static/js/impala/stats/chart.js

```diff
@@ -266,15 +266,15 @@
             function dayFormatter(d) { return Highcharts.dateFormat('%a, %b %e, %Y', new Date(d)); }
             function weekFormatter(d) { return format(gettext('Week of {0}'), Highcharts.dateFormat('%b %e, %Y', new Date(d))); }
             function monthFormatter(d) { return Highcharts.dateFormat('%B %Y', new Date(d)); }
-            function downloadFormatter(n) { return gettext(Highcharts.numberFormat(n, 0) + ' downloads'); }
-            function userFormatter(n) { return format(gettext('{0} users'), Highcharts.numberFormat(n, 0)); }
-            function addonsFormatter(n) { return format(gettext('{0} add-ons'), Highcharts.numberFormat(n, 0)); }
-            function collectionsFormatter(n) { return format(gettext('{0} collections'), Highcharts.numberFormat(n, 0)); }
-            function reviewsFormatter(n) { return format(gettext('{0} reviews'), Highcharts.numberFormat(n, 0)); }
+            function downloadFormatter(n) { return format(ngettext('{0} download', '{0} downloads', n), Highcharts.numberFormat(n, 0)); }
+            function userFormatter(n) { return format(ngettext('{0} user', '{0} users', n), Highcharts.numberFormat(n, 0)); }
+            function addonsFormatter(n) { return format(ngettext('{0} add-on', '{0} add-ons, n'), Highcharts.numberFormat(n, 0)); }
```

Comment:
Has anyone checked how these strings were exported?

I'm pretty sure this is missing a quote, resulting in `Z add-ons, n`.

Reply:
Oh my, I don't how I misplaced that `'`. I'll send a fix ASAP.

Reply:
Great.  If you don't have time to do it can you be sure to file an issue instead so it's not forgotten.

Reply:
@eviljeff Sent a PR.